project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc

```diff
@@ -630,9 +698,79 @@ function _c4m_user_notifications_new_content_published($node) {
 }
 
 /**
- * Email notification about existing content update at organic group.
+ * Helper function.
+ *
+ * Sends mail notification to group subscribers or content author and editors,
+ * when content is updated.
+ *
+ * @param object $node
+ *   Content node that was updated.
+ * @param string $users_group
+ *   Use 'group_followers' when notifying group followers about content update.
+ *   Use 'content_editors' when notifying content editors about content update.
  */
-function c4m_user_notifications_existing_group_content_updated($node) {
+function _c4m_user_notifications_notify_on_content_new_revision($node, $users_group) {
+  global $user;
+
+  switch ($users_group) {
+    case 'content_editors':
+      $message_type = 'c4m_notify_editors_content_updated';
+      $flag_suffix = 'node';
+      break;
+
+    case 'group_followers':
+      $message_type = 'c4m_content_updated';
+      $flag_suffix = 'group';
+      break;
+
+    default:
+      return;
+  }
+
+  $message = message_create($message_type);
+  // Define functions with which to resolve email template tokens.
+  c4m_message_load_tokens__c4m_content_inserted_or_updated($message);
+
+  $wrapper = entity_metadata_wrapper('message', $message);
+  $wrapper->field_node->set($node->nid);
+  $node_wrapper = entity_metadata_wrapper('node', $node);
+  $gid = $node_wrapper->{OG_AUDIENCE_FIELD}->value(array('identifier' => TRUE));
+  $wrapper->field_group_node->set($gid);
+  $wrapper->field_executing_user->set($user->uid);
+
+  // Construct the context, with parameters that will be used by
+  // c4m_user_notifications_message_subscribe_get_subscribers().
+  $context = array(
+    'entity_type' => $flag_suffix,
+  );
+
+  if ($users_group == 'group_followers') {
+    // Resolve comment body field, in teaser mode, and assign value to
+    // it's token.
+    $body_teaser = field_view_field('node', $node, 'c4m_body', 'teaser');
```

Comment:
The comment is talking about using the body field of the comment while in the code we take the body of the node.